README.md

```diff
@@ -63,33 +57,14 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be better if you follow Gitflow because you are not following Gitflow 😨 In Gitflow, your development branch will be the default branch and then you make your feature branch from the development branch and after getting approval on that feature branch you merge it to the development branch 👍